ui/StatusQ/tests/CMakeLists.txt

```diff
@@ -2,44 +2,48 @@ cmake_minimum_required(VERSION 3.5)
 
 project(TestStatusQ LANGUAGES CXX)
```

Comment:
Suggestion: it looks to me that this name did fit the scope of the project to configure StatusQ tests. Now it looks like TestHelpers would work better.

Reply:
Hmm, I think it's still ok because the project contains lib with testing utilities, executables for qml tests and c++ tests. So it's not only about helpers. Wdyt?

Reply:
`TestStatusQ` sounds to me like testing StatusQ lib but it's nothing.